dev/inc/assets.php

```diff
@@ -50,17 +50,36 @@ function wp_rig_scripts() {
 		)
 	);
 
-	// Enqueue skip-link-focus script.
-	wp_enqueue_script( 'wp-rig-skip-link-focus-fix', get_theme_file_uri( '/js/skip-link-focus-fix.js' ), array(), '20180514', false );
-	wp_script_add_data( 'wp-rig-skip-link-focus-fix', 'defer', true );
-
 	// Enqueue comment script on singular post/page views only.
 	if ( is_singular() && comments_open() && get_option( 'thread_comments' ) ) {
 		wp_enqueue_script( 'comment-reply' );
 	}
 }
 add_action( 'wp_enqueue_scripts', 'wp_rig_scripts' );
 
+/**
+ * Fix skip link focus in IE11.
+ *
+ * This does not enqueue the script because it is tiny and because it is only for IE11,
+ * thus it does not warrant having an entire dedicated blocking script being loaded.
+ *
+ * @link https://git.io/vWdr2
+ */
+function wp_rig_fix_skip_link_focus() {
+	if ( wp_rig_is_amp() ) {
+		return; // See <https://github.com/ampproject/amphtml/issues/18671>.
+	}
+	$skip_link_focus_fix_js_file = get_template_directory() . '/js/skip-link-focus-fix.js';
+	if ( ! file_exists( $skip_link_focus_fix_js_file ) ) {
+		_doing_it_wrong( __FUNCTION__, esc_html__( 'Unable to locate js/skip-link-focus-fix.js, perhaps due to not performing a build. If not desired, please unhook wp_rig_fix_skip_link_focus from the wp_print_footer_scripts.', '2.0' ) );
+		return;
+	}
+	echo '<script>';
+	echo file_get_contents( $skip_link_focus_fix_js_file );
```

Comment:
What do you think about inlining the minified version as done in https://github.com/WordPress/twentynineteen/pull/47?

Reply:
Let's wait until the issue there is resolved (unless it gets stuck forever), and then we can review the decided approach.